### PR TITLE
Feature/claudia 6280 regionlist

### PR DESCRIPTION
**Pull request by jesuspg**
#### Reviewers

@flopezag 
#### Description

add new operation
#### Testing

local

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -39,6 +39,30 @@ Final version aligned with FIWARE Release 5.2.1.
 
 # Group aiakos v1
 
+## Regions [/v1/support]
+
+Get the region list with keys
```
Comment (flopezag) [inline apiary.apib]: This is the description of the GET operation, not the description of the Regions resource, you should move it bellow line 47

File: apiary.apib [hunk 2]
```diff
@@ -39,6 +39,30 @@ Final version aligned with FIWARE Release 5.2.1.
...
+
+### Get list [GET]
+
```
Comment (flopezag) [inline apiary.apib:45]: You should describe the return/response values in order to clarify what does it mean a missign key-value (gpgkey, sshkey).

Besides, I prefer to have a more json-style approach, something like:

{
 regions: [
    {
      "regionName: "Spain2",
      "gpgkey": "2016-02-01T13:09:24.000Z",
      "sshkey": "2016-02-01T13:09:24.000Z"
    },
    {
      "regionName: "Trento2",
      "gpgkey": "2016-02-01T13:09:24.000Z"
    }
 ]
}

Comment (jesuspg) [reply apiary.apib:45]: as you can see, I'm using  a 'JSON.stringify(regionsHashTable);' to generate JSON

Comment (jesuspg) [reply apiary.apib:45]: if you need to return a json with more metadata (something like you propouse), I should add a function that iterate over the json, in order to generate a new json with your approach.

**Comment by flopezag** [comment]
LGTM

**Pull request merged**